tsfresh/feature_extraction/extraction.py

```diff
@@ -313,16 +314,21 @@ def _do_extraction(df, column_id, column_value, column_kind,
                                     function_kwargs=kwargs)
     distributor.close()
 
-    # Return a dataframe in the typical form (id as index and feature names as columns)
-    result = pd.DataFrame(result)
-    if "value" in result.columns:
-        result["value"] = result["value"].astype(float)
+    return_df_dict = defaultdict(dict)
```

Comment:
I would write a method for the conversion that now happens in lines 317-323 and test that method.

Reply:
Seems good, will do so.